.github/workflows/merge-gate.yml

```
@@ -0,0 +1,141 @@
+jobs:
+  gate-merge:
+    name: gate/merge
+    runs-on: ubuntu-latest
+    env:
+      REQUIRED_WORKFLOWS_JSON: '["Quality Gates","Build Gradle Public","Build Gradle Hybrid","Build Gradle Hybrid Docker","Build Maven Public"]'
+      OPTIONAL_WORKFLOWS_JSON: '["Security Reports"]'
+    steps:
+      - name: Resolve merge gate context
+        id: context
+        shell: bash
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)